_IBC/bitsong-axelar.json

```diff
@@ -0,0 +1,26 @@
+{
+  "$schema": "../ibc_data.schema.json",
+  "chain_1": {
+    "chain_name": "bitsong",
+    "client_id": "07-tendermint-82",
+    "connection_id": "connection-57"
+  },
+  "chain_2": {
+    "chain_name": "axelar",
+    "client_id": "07-tendermint-252",
+    "connection_id": "connection-243"
+  },
+  "channels": [
+    {
+      "chain_1_channel_id": "channel-38",
```

Comment:
Not quite following the schema. chain_1 would be an object, and channel_id would be a property within